cmd/minikube/cmd/start.go

```diff
@@ -156,6 +169,36 @@ func runStart(cmd *cobra.Command, args []string) {
 	clusterUp(&config)
 }
 
+// Set docker and oc cluster proxy in place
+func setProxy() {
+	if viper.IsSet("http-proxy") {
```

Comment:
I did try to find a resource which can actually validate user input for proxy URL but I didn't find any standard/non-standard way of handling it. url.Parse is something people use but it's not parse the proxy url which we want it to do.  I agree on the help string side changes.